src/Symfony/Bundle/FrameworkBundle/Command/WorkflowDumpCommand.php

```diff
@@ -39,13 +41,23 @@ protected function configure()
                 new InputArgument('name', InputArgument::REQUIRED, 'A workflow name'),
                 new InputArgument('marking', InputArgument::IS_ARRAY, 'A marking (a list of places)'),
                 new InputOption('label', 'l', InputArgument::OPTIONAL, 'Labels a graph'),
+                new InputOption('format', null, InputArgument::OPTIONAL, 'The dump format', 'dot'),
+                new InputOption(
+                    'transition',
+                    null,
+                    InputArgument::OPTIONAL,
```

Comment:
This is wrong:
* It's an option, not an arguement
* The value is required

Reply:
correct, fixed.